src/compiler.rs

```diff
@@ -332,6 +338,33 @@ impl<CM: CellManager, SSB: StepSelectorBuilder> Compiler<CM, SSB> {
         }
     }
 
+    fn compile_exposed<F, TraceArgs>(
+        &self,
+        ast: &astCircuit<F, TraceArgs>,
+        unit: &mut CompilationUnit<F>,
+    ) {
+        for forward_signal in ast.exposed.clone() {
+            let forward_placement = unit.placement.get_forward_placement(&forward_signal);
+            let exposed = (forward_placement.column, forward_placement.rotation);
+            unit.exposed.push(exposed);
+        }
+    }
```

Comment:
Note to self: I think we should add `expose` for shared signal as well.

Reply:
Yes, but later